Makefile

```diff
@@ -447,6 +452,14 @@ test/thrash_threads7: test/thrash_threads7.o libhts.a
 	$(CC) $(LDFLAGS) -o $@ test/thrash_threads7.o libhts.a -lz $(LIBS) -lpthread
 test_thrash: $(BUILT_THRASH_PROGRAMS)
 
+test/fieldarith.o: test/fieldarith.c $(htslib_sam_h)
```

Comment:
These dependencies are already listed about 30 lines above…

Reply:
Thanks for the spot.  I'm suspecting a rebase hiccup somewhere in the long history of this PR.  Should be gone soon.